sound/soc/sof/ipc4-topology.c

```diff
@@ -2690,8 +2690,8 @@ static enum sof_tokens common_copier_token_list[] = {
 	SOF_AUDIO_FMT_NUM_TOKENS,
 	SOF_IN_AUDIO_FORMAT_TOKENS,
 	SOF_OUT_AUDIO_FORMAT_TOKENS,
-	SOF_COPIER_DEEP_BUFFER_TOKENS,
-	SOF_COPIER_TOKENS,
+	SOF_COPIER_PREPARE_TOKENS,
+	SOF_COPIER_SETUP_PCM_TOKENS,
```

Comment:
I would leave the `SOF_COPIER_TOKENS` as it was, it applies to any type of copiers.
The `SOF_COPIER_DEEP_BUFFER_TOKENS` on the other hand is going to be repurposed to collect non dai side copier parameters, right?
`SOF_COPIER_PARAMETER_TOKENS` or `SOF_NON_DAI_COPIER_TOKENS`?

Reply:
I'll leave it as SOF_COPIER_PARAMETER_TOKENS, since we may in the future need to utilize it for DAI copiers too.